vsintegration/src/FSharp.Editor/FSharpKeywords.fs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+module Microsoft.VisualStudio.FSharp.Editor.FSharpKeywords
+  
+let keywordsWithDescription : (string * string) list =
+    [ "abstract",  """Indicates a method that either has no implementation in the type in which it is declared or that is virtual and has a default implementation."""
```

Comment:
All of these strings need to be localized.  My preference would be to place this table in ``lexhelp.fs`` with the help strings in ``FSComp.txt``, then this table is available to all users of FSharp.Compiler.Service